File: 2.0/python/texts/createTextAutoReply.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.texts.createTextAutoReply(
    body={
        'number': '67076',
        'keyword': 'CALLFIRE',
        'message': 'I am a leaf on the wind'
    }
).result()

print(response)

File: 2.0/php/texts/createTextAutoReply.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->createTextAutoReply();
        $request->getOperationConfig()->setHeaderParameters(array("Content-Type" => "application/json"));
        $body = '{
                    "number":"67076",
                    "keyword":"CALLFIRE",
                    "message":"I am a leaf on the wind"
                 }';
        $request->getOperationConfig()->setBodyParameter($body);
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
